admin/differ/Differ.scss

```diff
@@ -7,9 +7,13 @@
 
 .xh-admin-diff-detail {
   width: 700px;
+  max-height: 80vh;
   table {
+    display: flex;
     width: 100%;
     border-collapse: collapse;
+    overflow-y: scroll;
```

Comment:
This can be `overflow-y: auto` - that way, the scrollbar will only show and reserve space when the content is tall enough to scroll.